src/containers/adminDashboard/index.tsx

```diff
@@ -17,26 +19,27 @@ const EditUser = styled.div`
   width: 370px;
 `;
 
-const RoleDropdown = styled(Dropdown)<DropDownProps>`
+const RoleDropdown = styled(Select)`
   width: 100%;
   height: 36px;
 `;
 
-type DropdownItem = {
-  key: number;
-  label: string;
-};
-
 const AdminDashboard: React.FC = () => {
   const isSuperAdmin = true;
 
-  const roleList: DropdownItem[] = [
-    { key: 1, label: 'Standard user' },
-    { key: 2, label: 'Admin' },
-    { key: 3, label: 'Super admin' },
-  ];
+  const formRef = useRef<FormInstance>(null);
```

Comment:
Why this over useState?

Reply:
The other forms on the settings pages use form submission instead of having a callback that changes the state. I don't think it makes a difference functionally